docs-content/general/6_product-features/1_session-replay/performance-impact.md

```diff
@@ -28,3 +28,7 @@ Your client will send Highlight telemetry about every 3 seconds. We've taken ext
 1.  Only 1 request will be in-flight at a given time
 
 2.  Responsive to your end user's network speed
+
+## Session Replay
+
+Concerned about session replay impacting your web application? Read our blog post about it [here](https://highlight.io/blog/session-replay-performance).
```

Comment:
just made some last-minute tweaks to the [blog post google doc](https://docs.google.com/document/d/1F3hNX6lJgJmomQqRXo9MtaHLStYm7XB8zl3l70Io46Y/edit?usp=sharing) 

could we update the content here?

Reply:
Just pushed the updated content